pages/faq.md

```diff
@@ -0,0 +1,103 @@
+ <li>Stage Manager (SM): Responsible for communicating between performers backstage and run crew in house, as well as calling cues and handing off equipment.</li>
+ <li>Assistant Postions: These are assistant positions that are great for beginners to learn about a run crew position. Especially for large events, you may see aLD, aFoH and aTiC positions open where you can work side by side with a more experienced run crew member.</li>
+</ul>
+
+##### Do I need prior experience in event production?
+No, in fact, many of our members have little to no prior experience in event production when starting out. Just about everything can be learned on the job by participating in events and taking run crew positions. 
+
+##### When does AB Tech meet?
+AB Tech holds open weekly meetings for anyone interested in getting involved with upcoming events. Join our mailing list in the <a href="/join">Join</a> form for specific times and locations. We also meet to set up events throughout the week. For a list of upcoming events, please see our <a href="https://abtech.andrew.cmu.edu/tracker/" target="_blank">Tracker</a> site. 
+
+##### AB Tech does so many things in event production! How can I learn everything?
+Yep, this is a common dilemma for techies. Typically, members try out a few roles, eventually specializing in one or two roles and dabbling in the rest. Coming to events and trainings is the best way to work with equipment hands-on. You can also take an assistant position to be paired with a more experienced techie, who will teach you how to set up equipment and work a show.
+
+##### Is there a time commitment to being in AB Tech?
+Nope! You are free to sign up for run crew positions and come to setup/strike as your schedule allows. However, depending on the size of the event and run crew position, you may need to attend run crew meetings or prepare paperwork ahead of time. These meetings will be coordinated with the techie in charge as needed.
+
+##### What else do techies do?
+Sure, we work hard, but we also have a lot of fun together. After large events, we often get together for a celebratory meal off campus. We’ve also hosted game nights, karaoke, Fuku runs and ski trips for our members. If you’ve got an idea for a social event, let us know! 
+
+##### How do I get involved?
+Come talk to us! We are present at the FAIR and Winter FAIR - just look for the bright lights and follow the loud music. You can also join the mailing list by hitting the <a href="/join">Join</a> button above. See a person in a tech shirt? Ask them about the next event you can help at and they should hopefully be able to point you in the right direction.
+
+##### How else can I get connected?
+Join our <a href="https://www.facebook.com/cmuabtech" target="_blank">Facebook</a> page and follow us on <a href="https://www.instagram.com/cmuabtech/" target="_blank">Instagram</a> @cmuabtech to see all the cool things we’re doing!
+
+<h2>
```

Comment:
Change this h2 block to just be `## Requesting AB Tech`. You should be able to link to it as `#requesting-ab-tech`.